src/components/AttachmentPicker/index.native.js

```diff
@@ -142,18 +71,18 @@ class AttachmentPicker extends Component {
         this.menuItemData = [
             {
                 icon: Camera,
-                text: 'Take Photo',
-                pickAttachment: () => showImagePicker(RNImagePicker.launchCamera),
+                text: this.props.translate('attachmentPicker.takePhoto'),
+                pickAttachment: () => this.showImagePicker(RNImagePicker.launchCamera),
             },
             {
                 icon: Gallery,
-                text: 'Choose from Gallery',
-                pickAttachment: () => showImagePicker(RNImagePicker.launchImageLibrary),
+                text: this.props.translate('attachmentPicker.chooseFromGallery'),
+                pickAttachment: () => this.showImagePicker(RNImagePicker.launchImageLibrary),
             },
             {
                 icon: Paperclip,
-                text: 'Choose Document',
-                pickAttachment: showDocumentPicker,
+                text: this.props.translate('attachmentPicker.chooseDocument'),
+                pickAttachment: this.showDocumentPicker,
```

Comment:
`this` reference is lost here, even if `showDocumentPicker` is bound like

```js
 this.showDocumentPicker =  this.showDocumentPicker.bind(this)
```

Because `bind` happens at the end of the `constructor` and the assigned `pickAttachment` here would be the `showDocumentPicker` before `bind` runs

If the `if` condition in `showDocumentPicker` ever evaluates to true the app would silently log an error, and the user will not be notified that attachment picking failed (Though I don't know when that would happen, no storage permissions?)
![image](https://user-images.githubusercontent.com/12156624/121053401-fc471880-c7c3-11eb-9d76-bd075b5ac627.png)


The easiest fix would be to change this to:

```js
pickAttachment: () => this.showDocumentPicker(),
```